pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -176,6 +182,75 @@ func (o *OvirtMapper) MapDisks() map[string]cdiv1.DataVolume {
 	return dvs
 }
 
+func (o *OvirtMapper) mapNics() []kubevirtv1.Interface {
+	var kubevirtNics []kubevirtv1.Interface
+	nics, _ := o.vm.Nics()
+	for _, nic := range nics.Slice() {
+		// This network interface don't have any network specified.
+		if _, ok := nic.VnicProfile(); !ok {
+			continue
+		}
+
+		kubevirtNic := kubevirtv1.Interface{}
+		kubevirtNic.Name, _ = nic.Name()
```

Comment:
Aren't there any restrictions on the name format - like with the vm name? Shouldn't it be normalized?

Reply:
I am not sure, I will check.

Reply:
oVirt backend already implements the validation and only nic with alphanumeric characters and '-', '_' can be created. So I think we don't need the validation here.

Reply:
:+1: